fitsio/fitslib.py

```diff
@@ -1800,6 +1845,26 @@ def get_qmethod(qmethod):
     return _qmethod_map[qmethod]
 
 
+def get_dither_seed(dither_seed):
+    if dither_seed is None or dither_seed in ['clock', 'CLOCK']:
```

Comment:
```suggestion
    if dither_seed is None or (isinstance(dither_seed, str) and dither_seed.lower() == 'clock'):
```

Reply:
These are equivalent, and I actually think the first reads better

Reply:
actually no, I guess you are suggesting we accept things like `CloCk`

Reply:
I agree it is an odd edge case, but I can imagine someone putting in a single capitol by accident and getting super frustrated why things are broken / not working.